test/tests/functional/pbs_hook_debug_input.py

```diff
@@ -59,7 +59,12 @@ def remove_files_match(self, pattern):
         """
         for item in self.du.listdir(path=self.server_hooks_tmp_dir, sudo=True):
             if fnmatch.fnmatch(item, pattern):
-                self.du.rm(path=item, sudo=True, force=True)
+                self.du.rm(path=item, sudo=True)
+
+                #Check if the file was removed
```

Comment:
I think pep8 will complain to you about not having a space after the # mark.

Reply:
Thanks! Fixed it now.